package.json

```diff
@@ -16,9 +16,11 @@
     "cdk"
   ],
   "scripts": {
+    "start:dev": "concurrently \"npm run start\" \"npm run wait-on && npm run start:example\"",
```

Comment:
these are not needed, when example is launched you don't need to `npm start` on root